src/platform/packages/shared/kbn-scout/src/servers/start_servers.ts

```diff
@@ -38,7 +38,11 @@ export async function startServers(log: ToolingLog, options: StartServerOptions)
       procs,
       config,
       installDir: options.installDir,
-      extraKbnOpts: getExtraKbnOpts(options.installDir, config.get('serverless')),
+      extraKbnOpts: getExtraKbnOpts(
+        options.installDir,
+        config.get('serverless'),
+        options.kibanaConfig
+      ),
```

Comment:
I tested it locally and still wonder if it won't bring confusion: those `options.kibanaConfig` are not propagated into generated `.scout/servers/local.json` file. So I wonder what is the actual use case for it?
If it's only to start servers, but not to run tests, we need to explicitly mention it. Or maybe we should adjust logic to propagate those custom parameters into `local.json`?